src/main/haskell/language-kore/src/Data/Kore/AST/PureToKore.hs

```diff
@@ -67,8 +67,9 @@ extractPurePattern level p =
 -- Should be temporary measure
 sentencePureToKore
     :: MetaOrObject level => PureSentence level -> KoreSentence
-sentencePureToKore (SentenceAliasSentence (SentenceAlias a b c d)) =
-    constructUnifiedSentence SentenceAliasSentence $ SentenceAlias a b c d
+sentencePureToKore (SentenceAliasSentence (SentenceAlias a b c d e f)) =
```

Comment:
Since you're not simply passing all the arguments in bulk to another function, I'd consider naming them, with or without using full record matching and rebuilding.